R/preparedata.R

```diff
@@ -1592,6 +1592,61 @@ setAxisTitles <- function(x, chart.type, drop, values.title = "")
     x
 }
 
+#' Helps tidy Q variables and tables
+#' @description Inputs supplied via input.data.raw can be in a range of
+#'  formats. This function does a minimal job of checking for attribute
+#'  and using these as names when appropriate.
+#'
```

Comment:
Could add more detail to this description. e.g "returns raw data early if the raw data is a specific structure. If the span attribute is requested, then the rows of span attribute is returned"

Reply:
I've added more detail to the description in 65bde8f